src/indices/indexed-class.lisp

```diff
@@ -247,12 +250,12 @@ also index subclasses of the class to which the slot belongs, default is T")
 
 ;;; avoid late instantiation
 
-#+(or allegro cmu openmcl sbcl)
+#+(or allegro cmu openmcl sbcl lispworks)
```

Comment:
I forget the exact error, but I needed this to get things moving. I'm assuming you'll look at this and go "yeah, that makes sense". If it doesn't make sense, then I'll try to pull out the actual error message that I got.